source/unified-test-format/unified-test-format.rst

```diff
@@ -298,9 +323,12 @@ The top-level fields of a test file are as follows:
 - ``initialData``: Optional array of one or more `collectionData`_ objects. Data
   that will exist in collections before each test case is executed.
 
-  Before each test and for each `collectionData`_, the test runner MUST drop the
-  collection and insert the specified documents (if any) using a "majority"
-  write concern. If no documents are specified, the test runner MUST create the
+  Before each test and for each `collectionData`_, the test runner MUST drop
+  the collection. If a ``collectionOptions`` document is present, the test
+  runner MUST execute a ``create`` command to create the collection with the
+  specified options. The test runner MUST then insert the specified documents
```

Comment:
We may want to be explicit here that the `create` should be run with a "majority" write concern. Consider:
```suggestion
  specified options and a "majority" write concern. The test runner MUST then insert the specified documents
```

Reply:
I realized the information about creating collections is actually duplicated here and in the `Executing a Test` section. I removed it from here and updated the other one instead.